src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -219,7 +221,7 @@ protected override void OnActiveStateChanged()
         }
 
         public event Action<Model3D> RequestAttachToScene;
-        protected void OnRequestAttachToScene(Model3D model3D)
+        protected virtual void OnRequestAttachToScene(Model3D model3D)
```

Comment:
unfortunately this method will likely be removed / unused in the helix-upgrade branch - it might be possible to retain it and redirect it to do something useful with the new helix api though.

Reply:
Will this affect us when we cherry pick this test into helix-upgrade now?

To build this as a unit test I just needed an overridable method called from `AggregateRenderPackages`. I guess I could use another method with these characteristics in helix-upgrade if it exists, or I could add this one with no behaviour if not. Does that sound ok?